### ISSUE TITLE
Hybrid Forecast Model with TFT

### ISSUE THREAD

**Issue by polal2is** [jdb78/pytorch-forecasting#397]
Hi,
First of all thank you very much for this remarkable piece of work!

I don't know whether it is supported but there are use cases where forecasting time varying reals and time varying categories at the same times turns useful. For example predicting next events in a sequence and their durations.

How would such hybrid targets be handled in terms of loss and say target_normalizer? Shall we pass lists such as for example: 
loss= [CrossEntropy(), QuantileLoss()]? Didn't seem to work on first attempt.

Many thanks in advance!

**Comment by jdb78** [comment]
You have to use the [MultiLoss](https://pytorch-forecasting.readthedocs.io/en/latest/api/pytorch_forecasting.metrics.MultiLoss.html). But this is a good point. We should automatically convert a list to a MultiLoss.

**Comment by polal2is** [comment]
Thank you very for pointing this out ! I'll give it a go then.
